### PR TITLE
Add api name and version headers

### PR DESCRIPTION
**Pull request by richardartoul**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
+import os
 from setuptools import setup
 
 
-version = "0.11.1"
+VERSION_FILE = open(os.path.join("hyperdash", 'VERSION'))
+VERSION = VERSION_FILE.read().strip()
```
Comment (spence) [inline setup.py]: Use consistent quote style.

Mind using `with` to ensure it's closed properly?
```python
with open(os.path.join('hyperdash', 'VERSION')) as f:
    VERSION = f.read().strip()
```

File: hyperdash_cli/cli.py [hunk 2]
```diff
@@ -281,6 +283,10 @@ def stderr_loop():
     wrapped()
 
 
+def version(args=None):
+    print("hyperdash {}".format(get_hyperdash_version()))
```
Comment (spence) [inline hyperdash_cli/cli.py]: Omit `args=None`?

File: hyperdash/constants.py [hunk 3]
```diff
@@ -62,3 +68,8 @@ def get_hyperdash_json_local_path():
     if not local_path:
         return None
     return os.path.join(local_path, "hyperdash.json")
+
+
+def get_hyperdash_version():
+    """Return the version of Hyperdash as a string."""
+    return open(os.path.join(os.path.dirname(__file__), 'VERSION')).read().strip()
```
Comment (spence) [inline hyperdash/constants.py]: Same comment as before. Use `with`.

File: hyperdash/constants.py [hunk 4]
```diff
@@ -6,11 +6,17 @@
...
+API_KEY_NAME = "x-hyperdash-api"
 HTTP_ENDPOINT = "/api/v1/sdk/http"
 CACHE_API_KEY_FOR_SECONDS = 300
 # 20 KiB
 MAX_LOG_SIZE_BYTES = 20480
 
+API_NAME_MONITOR = "monitor"
+API_NAME_EXPERIMENT = "experiment"
+API_NAME_CLI_RUN = "cli_run"
+API_NAME_JUPYTER = "jupyter"
```
Comment (spence) [inline hyperdash/constants.py]: Single quote or double quote?

**Comment by spence** [comment]
Otherwise LGTM

File: hyperdash_cli/cli.py (hunk 2 above)
Comment (andrewschreiber) [reply hyperdash_cli/cli.py]: I think it's fine - emphasizes the method is connected to line 414

Comment (spence) [reply hyperdash_cli/cli.py]: Looking at the other methods here, `run` is the only one that receives `args`, so it feels weird that `run` is not the only function with `args` in the function declaration.

**Pull request merged**